airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.py

```diff
@@ -29,57 +29,42 @@ class Config:
 
     name: str = Field(
         title="Name",
-        description="The name value of insight",
     )
 
     fields: Optional[List[ValidFields]] = Field(
         title="Fields",
-        description="A list of chosen fields for fields parameter",
         default=[],
     )
 
     breakdowns: Optional[List[ValidBreakdowns]] = Field(
         title="Breakdowns",
-        description="A list of chosen breakdowns for breakdowns",
         default=[],
     )
 
     action_breakdowns: Optional[List[ValidActionBreakdowns]] = Field(
         title="Action Breakdowns",
-        description="A list of chosen action_breakdowns for action_breakdowns",
         default=[],
     )
 
     time_increment: Optional[PositiveInt] = Field(
         title="Time Increment",
-        description=(
-            "Time window in days by which to aggregate statistics. The sync will be chunked into N day intervals, where N is the number of days you specified. "
-            "For example, if you set this value to 7, then all statistics will be reported as 7-day aggregates by starting from the start_date. If the start and end dates are October 1st and October 30th, then the connector will output 5 records: 01 - 06, 07 - 13, 14 - 20, 21 - 27, and 28 - 30 (3 days only)."
-        ),
```

Comment:
should we ask doc team to re-write it? seems like a crucial piece of info at a first glance